src/components/ScoreBoard/partials/TeamScore.tsx

```diff
@@ -6,7 +6,7 @@ type TeamScoreProps = {
 
 const TeamScore: React.FC<TeamScoreProps> = ({ name }) => {
   const [score, setScore] = React.useState(
-    parseInt(localStorage.getItem(name)) || 0
+    parseInt(localStorage.getItem(name) || "0")
```

Comment:
these changes are already in main I believe. Just do a rebase with main before you merge